### ISSUE TITLE
How cool would it be to have a calculator app that match the OSX look and feel

### ISSUE THREAD

**Issue by aurelienmaury** [helloSystem/Utilities#186]

**Comment by Hierosme** [comment]
Sorry for the noise.

That request is for tracking ...

Still FixedSize but with a size management
Spanning col/row for 0 and =
Introduse memory line MC, M+, M-, MR
Introduse ± key
Fit with layout of OsX
Use true operator symbol and convert them before the eval
Add CHANGELOGSD file

**Comment by louies0623** [comment]
I agree, I wish it had an interface like this too, and emulated buttons.
![pcalc38](https://github.com/helloSystem/Utilities/assets/44593430/530dd725-0a96-4e9f-96fe-8e75822b1570)

Maybe give it a geek mode.
![GeekCalc](https://github.com/helloSystem/Utilities/assets/44593430/66fef603-f740-4ae1-b5fe-98d869e3fa03)

**Comment by Hierosme** [comment]
It look like we go for create a custum class for our button.

I like a lot PCalc
![image](https://github.com/helloSystem/Utilities/assets/56162977/3f131e9b-8b28-4450-9a17-42c5c1db49df)

But a realistic step is more like that
![image](https://github.com/helloSystem/Utilities/assets/56162977/b0af6cb5-39d8-4d97-bc5f-6332eaf143ca)

The better is to migrate the Calculator.app to a True .ui file, just for have better control when will have a Scientific Calcultaor layout to display.
